drivers/npmx/shell/buck.c

```diff
@@ -76,6 +76,26 @@ static int cmd_buck_active_discharge_get(const struct shell *shell, size_t argc,
 	return 0;
 }
 
+static npmx_buck_gpio_t buck_gpio_index_convert(int32_t gpio_idx)
```

Comment:
Please rename it to `gpio_index`. Try not to use any short names when it is possible.